sklearn/multioutput.py

```diff
@@ -513,15 +513,15 @@ class MultiOutputClassifier(ClassifierMixin, _MultiOutputEstimator):
     def __init__(self, estimator, *, n_jobs=None):
         super().__init__(estimator, n_jobs=n_jobs)
 
-    def fit(self, X, Y, sample_weight=None, **fit_params):
+    def fit(self, X, y, sample_weight=None, **fit_params):
```

Comment:
I agree with the change, but we need to go through a deprecation, for people who pass `Y` as a keyword argument.